### PR TITLE
[Edit]: Python math-module term entries

### PR DESCRIPTION
**Pull request by ishg-153**
### Description
Added separate shell blocks for examples in all the existing term entries.

### Type of Change

- Adding a new entry
- Editing an existing entry (fixing a typo, bug, issues, etc)
- Updating the documentation

### Checklist

- [x] All writings are my own.
- [x] My entry follows the Codecademy Docs style guide.
- [x] My changes generate no new warnings.
- [x] I have performed a self-review of my own writing and code.
- [x] I have checked my entry and corrected any misspellings.
- [x] I have made corresponding changes to the documentation if needed.
- [x] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x] I have linked any issues that are relevant to this PR in the `Issues Solved` section.

### CODE REVIEW

**Comment by ishg-153** [comment]
Hey @dakshdeepHERE Are you done reviewing this?

**Comment by dakshdeepHERE** [comment]
Hey @ishg-153, yep I will review it today.

**Review by dakshdeepHERE — CHANGES_REQUESTED** [review]
Few changes from my side overall looks perfect.

File: content/python/concepts/math-module/terms/math-sqrt/math-sqrt.md [hunk 1]
```diff
@@ -22,38 +22,46 @@ math.sqrt(x)
...
 ```python
 import math
 
 print(math.sqrt(5))
-
-# Output: 2.23606797749979
 ```
 
-## Example 2
+The above code give the following output:
```
Comment (ishg-153) [inline content/python/concepts/math-module/terms/math-sqrt/math-sqrt.md:35]:
```suggestion
The above code gives the following output:
```

File: content/python/concepts/math-module/terms/math-sqrt/math-sqrt.md [hunk 2]
```diff
@@ -22,38 +22,46 @@ math.sqrt(x)
...
 Use `math.sqrt()` to return the square root of the float `5.5`:
 
 ```python
 import math
 
 print(math.sqrt(5.5))
+```
 
-# Output: 2.345207879911715
+The above code give the following output:
```
Comment (ishg-153) [inline content/python/concepts/math-module/terms/math-sqrt/math-sqrt.md:49]:
```suggestion
The above code gives the following output:
```

File: content/python/concepts/math-module/terms/math-tan/math-tan.md [hunk 3]
```diff
@@ -28,25 +28,31 @@ Use `math.tan()` to return the tangent of the integer `5` measured in radians:
 import math
 
 print(math.tan(5))
-
-# Output: -3.380515006246586
 ```
 
-## Example 2
+The above code give the following output:
```
Comment (ishg-153) [inline content/python/concepts/math-module/terms/math-tan/math-tan.md:33]:
```suggestion
The above code gives the following output:
```

File: content/python/concepts/math-module/terms/math-tan/math-tan.md [hunk 4]
```diff
@@ -28,25 +28,31 @@ Use `math.tan()` to return the tangent of the integer `5` measured in radians:
...
 Use `math.tan()` to return the tangent of the float `5.5` measured in radians:
 
 ```python
 import math
 
-print(math.tan(5.5))
+print(math.tan(5.5)) 
+```
+
+The above code give the following output:
```
Comment (ishg-153) [inline content/python/concepts/math-module/terms/math-tan/math-tan.md:47]:
```suggestion
The above code gives the following output:
```

**Comment by ishg-153** [comment]
@dakshdeepHERE Sorry for the delay on my part, made the change everywhere!

**Review by Sriparno08 — APPROVED** [review]
Looks good 👍

**Pull request merged**